io_scene_godot/converters/mesh.py

```diff
@@ -25,44 +24,49 @@ def export_mesh_node(escn_file, export_settings, obj, parent_gd_node):
         parent_gd_node = physics.export_physics_properties(
             escn_file, export_settings, obj, parent_gd_node
         )
-
-    if physics.has_physics(obj) and obj.display_type == "WIRE":
         # skip wire mesh which is used as collision mesh
-        return parent_gd_node
+        if obj.display_type == "WIRE":
+            return parent_gd_node
 
     mesh_node = NodeTemplate(obj.name, "MeshInstance", parent_gd_node)
-    mesh_exporter = MeshResourceExporter(obj)
+    mesh_exporter = ArrayMeshResourceExporter(obj)
 
-    armature_data = get_modifier_armature_data(obj)
-    if ("ARMATURE" in export_settings['object_types'] and
-            armature_data is not None):
-        skeleton_node = armature.find_skeletion_node(parent_gd_node)
-        if skeleton_node is not None:
-            mesh_exporter.init_mesh_bones_data(skeleton_node)
-            mesh_node['skeleton'] = NodePath(
-                mesh_node.get_path(), skeleton_node.get_path())
+    armature_obj = None
+    if "ARMATURE" in export_settings['object_types']:
+        armature_obj = get_modifier_armature(obj)
+        if armature_obj:
+            mesh_exporter.init_mesh_bones_data(armature_obj, export_settings)
+            # set armature to REST so current pose does not affect converted meshes.
+            armature_pose_position = armature_obj.data.pose_position
+            armature_obj.data.pose_position = "REST"
 
     mesh_id = mesh_exporter.export_mesh(escn_file, export_settings)
 
-    mesh_node['mesh'] = "SubResource({})".format(mesh_id)
-    mesh_node['visible'] = obj.visible_get()
+    if armature_obj:
```

Comment:
if "ARMATURE" not in export_settings['object_types', there is a risk of `armature_pose_position` being uninitialized